tests/transports-governance/test_helpers.py

```
@@ -0,0 +1,644 @@
+            return [{"role": "user", "content": content}]
+
+        else:
+            return [{"role": "user", "content": fake.sentence()}]
+
+    @staticmethod
+    def make_test_request(
+        client: GovernanceTestClient,
+        vk_value: str,
+        model: str = "gpt-3.5-turbo",
+        max_tokens: int = 50,
+        **kwargs,
```

Ruff (0.12.2)

Missing type annotation for `**kwargs`

(ANN003)